sherpa/astro/tests/test_astro_plot.py

```diff
@@ -195,7 +254,7 @@ def test_sourceplot_counts(caplog, make_basic_datapha):
         sp.prepare(data, src)
 
     assert len(caplog.records) == 0
-    check_sourceplot_energy(sp)
+    check_sourceplot_energy(sp, rate=False)
```

Comment:
@hamogu - here is a case when `check_sourceplot_energy` is called with `rate=False`.